text/0005-application-formalization.md

```diff
@@ -0,0 +1,196 @@
+| Property | Type    | Description                                                          | Notes                            |
+| ---------| ------- | -------------------------------------------------------------------- | -------------------------------- |
+| id       | String  | Identifier for the module conforming to pattern: `{name}-{version}`  | e.g. "mod-orders-storage-13.5.0" |
+| name     | String  | Name of the module                                                   | e.g. "mod-orders"                |
+| version  | String  | Version of the module                                                | e.g. "13.5.0"                    |
+
+**ModuleId Properties**
+| Property | Type    | Description                                                          | Notes                            |
+| -------- | ------- | -------------------------------------------------------------------- | -------------------------------- |
+| id       | String  | Identifier for the module conforming to pattern: `{name}-{version}`  | e.g. "mod-orders-storage-13.5.0" |
+| name     | String  | Name of the module                                                   | e.g. "mod-orders"                |
+| version  | String  | Version of the module                                                | e.g. "13.5.0"                    |
+| url      | String  | URL pointing to the module descriptor for this module                | e.g. "https://folio-registry.dev.folio.org/_/proxy/modules/mod-orders-storage-13.5.0" |
+
+Example:
+```
+{
+  "id": "app-gobi-0.0.1",
+  "name": "app-gobi",
+  "version": "0.0.1",
+  "description": "Application facilitating the placement of orders via GOBI and other compatible vendors",
+  "dependencies": [
+    {
+      "id": "app-orders-0.0.1",
+      "name": "app-orders",
+      "version": "^0.0.1"
+    },
+    {
+      "id": "app-organizations-0.0.1",
+      "name": "app-organizations",
+      "version": "^0.0.1"
+    }
+  ],
+  "modules": [
+    {
+      "id": "mod-gobi-2.6.0",
+      "name": "mod-gobi",
+      "version": "2.6.0",
+	  "url": "https://folio-registry.dev.folio.org/_/proxy/modules/mod-gobi-2.6.0"
+    }
+  ],
+  "uiModules": [
+    {
+      "id": "folio_gobi-settings-2.0.0",
+      "name": "folio_gobi-settings",
+      "version": "2.0.0",
+	  "url": "https://folio-registry.dev.folio.org/_/proxy/modules/folio_gobi-settings-2.0.0"
+    }
+  ]
+}
+```
+**N.B.** Full module descriptors and metadata have been omitted for readability reasons.
+
+## Benefits
+* This is the first step toward breaking the monolithic release into application-specific releases
+* Results in fewer things to work with at the system operator level
+* Brings us closer to realizing the idea of having an application store/marketplace
+* Allows for introducing/expanding the bounded context (subject of a separate RFC)
+
+## Risks and Drawbacks
+* Folio has too many module dependencies which must be reduced
+* Transition to Applications will need to be iterative
```

Comment:
Why? Is this approach required? Why can the formalization start with smaller Applications instead?

Reply:
It's not required, but we see this as the most practical way to do it.
```suggestion
* Transition to Applications should be iterative
```